source/support/communicating-with-users/common-responses.html.md

```diff
@@ -95,6 +95,12 @@ Emails offering feedback or suggestions on government policy or politicians shou
 
 ## Latest release questions
 
+### When is the next breaking release happening
+
+Our current plan is to wait until we've updated all the components for the brand refresh before releasing the next breaking release of GOV.UK Frontend, which we're expecting to take us until close to the end of the year.
+
+We're aware the longer it takes, the more deprecated features we may have to remove in that breaking release (or default enabling of features like for the File upload), so we're trying to keep an eye on that and may revisit this as we review our roadmap.
```

Comment:
Unfamiliar with the File upload stuff so I'm a tad confused by "or default enabling of features like for the File upload". Does that mean we enabled some new features by default where they didn't override an old setting?

Reply:
Oh, is it saying File Upload features and the like would only be enabled by default at the major release?

Reply:
Cheers for spotting that, I'd drafted this to answer a question that was linked to the File upload (whose `javascript` option will be enabled by default in 6.0.0) and forgot to clean it up. I've commited your suggestion.